### PR TITLE
tests: completes test views

### LINKED ISSUE #804: Complete test views
The file https://github.com/inspirehep/inspire-next/blob/master/tests/test_views.py currently contains tests for the `/detailed` pages. 

Tests should be added for other main views such as:

* [ ] Landing pages (/literature, /jobs, ...)
* [ ] /admin/holdingpen/list (shall we create some objects and make sure they appear in the list ?)
* [ ] /author/new
* [ ] /author/update?recid=997876 ( so that imports data from that recid, otherwise it redirects to /new )
* [ ] /doi/search
* [ ] /arxiv/search
* [ ] /search with some different searches, sorting options, ...
* [ ] /submit/literature/create

Any others?

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**
* Closes #804.

Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>

### CODE REVIEW

File: tests/test_views.py [hunk 1]
```diff
@@ -24,13 +24,88 @@
 
 class ViewsTests(InvenioTestCase):
     """Tests views."""
+    def setUp(self):
+        self.login('admin', 'admin')
```
Comment (jalavik) [inline tests/test_views.py]: I suspect the login/logout part may slow down the tests a bit, so if you do not need it for every test then we can just call it in the few tests that do.

**Comment by ioannistsanaktsidis** [comment]
cc @jalavik

**Comment by jmartinm** [comment]
It seemed to work with the changes to `install.sh` right? cc @jalavik

**Comment by jalavik** [comment]
Yeah, it does. Great. @ioannistsanaktsidis So you can remove the commented out parts completely and is the plan to keep the detailed record check uncommented?

**Comment by jmartinm** [comment]
> Yeah, it does. Great. @ioannistsanaktsidis So you can remove the commented out parts completely and is the plan to keep the detailed record check uncommented?

Yes, I guess `test_detailed_records` was commented out to speed up this test? I think it should be left in the tests (although it is a bit slow)

**Comment by ioannistsanaktsidis** [comment]
Exactly the test_detailed was commented for speed reasons. I will amend the commit and we are good to go.

**Pull request merged**